components/console/introduction.rst

```diff
@@ -296,9 +296,11 @@ declare a one-letter shortcut that you can call with a single dash like
 
 .. tip::
 
-    It is also possible to make an option *optionally* accept a value (so that
-    ``--yell``, ``--yell=loud`` or ``--yell loud`` work). Options can also be configured to
-    accept an array of values.
+    There is nothing forbidding you to create a command with an *optional option*,
```

Comment:
-1 on the term "optional option", options are always optional. It's the value we are talking about here